api/GestureEvent.json

```diff
@@ -0,0 +1,259 @@
+{
+  "api": {
+    "GestureEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/GestureEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": "9"
+          },
+          "safari_ios": {
+            "version_added": "2"
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": false,
+          "deprecated": false
+        }
+      },
+      "GestureEvent": {
```

Comment:
I'm not finding the page or table corresponding to the constructor :/ Where is this phantom page? :)

Reply:
Hmm I've not come across too many of these but more so in this last % of the data migration. Not being certain with what's happened before, I added it with "basic" data to reflect the root page as currently displayed.